# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/ipc/PAPZCTreeManager.ipdl
# Commit: da8c34fec36e
# Full Hash: da8c34fec36e933ccfe93e458ccdbe9328132897
# Author: Mike Conley <mconley@mozilla.com>
# Date: 2022-06-22 03:32:25
# Regressor Bug: 1773865
# File Overlap Count: 1
# Description:
#   Bug 1773865 - Dispatch an event on the window document when a pinch zoom gesture ends. r=botond,smaug,NeilDeakin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D149283
# ==============================================================================

diff -r 2510b654e8f2 -r da8c34fec36e gfx/layers/ipc/PAPZCTreeManager.ipdl
--- a/gfx/layers/ipc/PAPZCTreeManager.ipdl	Tue Jun 21 13:56:53 2022 +0000
+++ b/gfx/layers/ipc/PAPZCTreeManager.ipdl	Tue Jun 21 14:02:19 2022 +0000
@@ -83,6 +83,8 @@
                            Modifiers aModifiers);
 
   async CancelAutoscroll(ViewID aScrollId);
+
+  async NotifyScaleGestureComplete(ViewID aScrollId, float aScale);
 };
 
 } // namespace gfx